chrome/locale/en-US/zotero/zotero.dtd

```diff
@@ -272,8 +272,10 @@
 <!ENTITY zotero.feedSettings.title.label                "Title:">
 <!ENTITY zotero.feedSettings.refresh.label1             "Update feed every">
 <!ENTITY zotero.feedSettings.refresh.label2             "hour(s)">
-<!ENTITY zotero.feedSettings.cleanupAfter.label1         "Remove read articles after ">
+<!ENTITY zotero.feedSettings.cleanupAfter.label1         "Remove articles after ">
 <!ENTITY zotero.feedSettings.cleanupAfter.label2         "day(s)">
+<!ENTITY zotero.feedSettings.cleanupReadAfter.label1         "Remove read articles after ">
+<!ENTITY zotero.feedSettings.cleanupReadAfter.label2         "day(s)">
```

Comment:
While we're here, we don't actually call anything "articles" in Zotero, and it's not necessarily accurate. I think can just call them "feed items". ("feed" is a little redundant, but maybe worth being clear that we're not talking about any other items.)